docs/about/index.md

```diff
@@ -0,0 +1,86 @@
+.NET Documentation
+==================
+
+Welcome to .NET! You can build any kind of application you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next application today, on Windows, OS X and Linux. This documentation shows you how.
+
+Languages
+---------
+
+.NET is multi-language as a design point. It supports multiple languages and enables interop between them. C#, F# and Visual Basic are the most popular. C# and F# are both multi-paradigm, with C# being primarily object-oriented and F# primarily functional. Visual Basic is the most productive and easiest to learn. The following C# example demonstrates the most basic console app.
```

Comment:
"multi-language as a design point" sounds like jargon.

How about ".NET is a multi-language programming platform."